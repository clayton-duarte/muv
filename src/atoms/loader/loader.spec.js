import React from 'react';

import Loader from '.';

it('should render a loader', () => {
  const wrapper = mount(
    <Loader />,
  );
  expect(wrapper).toMatchSnapshot();
});
