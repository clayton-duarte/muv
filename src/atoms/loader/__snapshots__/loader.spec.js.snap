// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a loader 1`] = `
<Loader>
  <styled.p
    theme={
      Object {
        "color": Object {
          "black": "#222222",
          "darkgray": "#515151",
          "error": "#dd3333",
          "gray": "#a9abae",
          "lightgray": "#e5e5e5",
          "overlay": "rgba(0, 0, 0, .15)",
          "primary": "#0081c8",
          "secondary": "#00245f",
          "success": "#33bb33",
          "warn": "#dd3333",
          "white": "#fefefe",
        },
        "font": Object {
          "color": "#222222",
          "family": "\\"Ubuntu\\", sans-serif",
          "size": "16px",
          "spacing": 0,
        },
        "shape": Object {
          "border": ".125rem solid rgba(0, 0, 0, .15)",
          "float": "0 .5rem .5rem 0 rgba(0, 0, 0, .15)",
          "margin": ".25rem .125rem",
          "padding": ".75rem .875rem",
          "radius": ".5rem",
          "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
          "size": "16px",
        },
        "transition": Object {
          "time": ".3s ease",
        },
      }
    }
  >
    <StyledComponent
      forwardedClass={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bdVaJa",
            "isStatic": false,
            "lastClassName": "vSNlF",
            "rules": Array [
              "
color: ",
              [Function],
              ";
",
            ],
          },
          "defaultProps": Object {
            "theme": Object {
              "color": Object {
                "black": "#222222",
                "darkgray": "#515151",
                "error": "#dd3333",
                "gray": "#a9abae",
                "lightgray": "#e5e5e5",
                "overlay": "rgba(0, 0, 0, .15)",
                "primary": "#0081c8",
                "secondary": "#00245f",
                "success": "#33bb33",
                "warn": "#dd3333",
                "white": "#fefefe",
              },
              "font": Object {
                "color": "#222222",
                "family": "\\"Ubuntu\\", sans-serif",
                "size": "16px",
                "spacing": 0,
              },
              "shape": Object {
                "border": ".125rem solid rgba(0, 0, 0, .15)",
                "float": "0 .5rem .5rem 0 rgba(0, 0, 0, .15)",
                "margin": ".25rem .125rem",
                "padding": ".75rem .875rem",
                "radius": ".5rem",
                "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
                "size": "16px",
              },
              "transition": Object {
                "time": ".3s ease",
              },
            },
          },
          "displayName": "styled.p",
          "render": [Function],
          "styledComponentId": "sc-bdVaJa",
          "target": "p",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        }
      }
      forwardedRef={null}
      theme={
        Object {
          "color": Object {
            "black": "#222222",
            "darkgray": "#515151",
            "error": "#dd3333",
            "gray": "#a9abae",
            "lightgray": "#e5e5e5",
            "overlay": "rgba(0, 0, 0, .15)",
            "primary": "#0081c8",
            "secondary": "#00245f",
            "success": "#33bb33",
            "warn": "#dd3333",
            "white": "#fefefe",
          },
          "font": Object {
            "color": "#222222",
            "family": "\\"Ubuntu\\", sans-serif",
            "size": "16px",
            "spacing": 0,
          },
          "shape": Object {
            "border": ".125rem solid rgba(0, 0, 0, .15)",
            "float": "0 .5rem .5rem 0 rgba(0, 0, 0, .15)",
            "margin": ".25rem .125rem",
            "padding": ".75rem .875rem",
            "radius": ".5rem",
            "shadow": "0 .25rem .25rem 0 rgba(0, 0, 0, .15)",
            "size": "16px",
          },
          "transition": Object {
            "time": ".3s ease",
          },
        }
      }
    >
      <p
        className="sc-bdVaJa vSNlF"
      >
        ...
      </p>
    </StyledComponent>
  </styled.p>
</Loader>
`;
